extensions/amp-story/1.0/amp-story-quiz.css

```diff
@@ -0,0 +1,145 @@
+@import url('https://fonts.googleapis.com/css?family=Poppins&display=swap');
+
```

Comment:
Please add RTL languages support. Since the `dir` property won't pierce the shadow boundaries, you need to propagate the direction yourself. There's an example of how it's done here: https://github.com/ampproject/amphtml/blob/master/extensions/amp-story/1.0/amp-story-consent.js#L293

Reply:
Done